packages/application-components/src/components/modal-pages/internals/modal-page.tsx

```diff
@@ -59,6 +59,7 @@ type Props = {
   baseZIndex: number;
   getParentSelector: typeof getDefaultParentSelector;
   shouldDelayOnClose: boolean;
+  afterOpenStyles?: string | { [key: string]: string | number };
```

Comment:
```suggestion
  afterOpenStyles?: string | CSSProperties;
```

Reply:
Using `CSSProperties` returns the following error on line 125:
```
 Overload 1 of 2, '(template: TemplateStringsArray, ...args: CSSInterpolation[]): string', gave the following error.
    Argument of type 'CSSProperties | SerializedStyles' is not assignable to parameter of type 'TemplateStringsArray'.
      Type 'PropertiesFallback<string | number, string & {}>' is missing the following properties from type 'TemplateStringsArray': raw, length, concat, join, and 18 more.
  Overload 2 of 2, '(...args: CSSInterpolation[]): string', gave the following error.
    Argument of type 'CSSProperties | SerializedStyles' is not assignable to parameter of type 'CSSInterpolation'.
      Type 'CSSProperties' is not assignable to type 'CSSInterpolation'.
        Type 'PropertiesFallback<string | number, string & {}>' is not assignable to type 'CSSObject'.
```

Using `CSSObject` like so works for typing the prop:
```
import { CSSObject } from '@emotion/react';

...
afterOpenStyles?: string | CSSObject;
```

Reply:
Ah yes, thanks!